proposals/0376-function-back-deployment.md

```diff
@@ -9,6 +9,10 @@
 
 ## Introduction
 
+This proposal introduces a `@backDeployed` attribute to allow ABI-stable libraries to extend the effective availability of their own public APIs to older OSes, where the API isn't actually present in the library that shipped with those OSes. While this resembles a polyfill mechanically, it does not help a module to overlay a polyfill on top of an API from another module.
```

Comment:
`While this resembles a polyfill mechanically, it does not help a module to overlay a polyfill on top of an API from another module.`

This sentence feels a little out of place to me in the introduction section. Maybe it ought to go in the Future Directions section and be expanded upon a bit? That section could discuss in greater detail the use case of providing polyfills for other module's APIs.

Reply:
My takeaway from the discussion is that this clarification is very much needed as a headline sentence: I don't think people will or should need to read down to the future directions to learn that this isn't a polyfill feature.

Reply:
Fair enough! I just think it needs wordsmithing to be part of the Introduction if we want to include it there. The current wording (I think taken verbatim from my forum post?) feels to me like an extremely abrupt introduction of the concept of "polyfill" and needs a bit more elaboration.